packages/common/src/messages/sign-on/pages.ts

```diff
@@ -72,7 +71,7 @@ export const pickHandlePageMessages = {
 export const finishProfilePageMessages = {
   header: 'Finish Your Profile',
   description:
-    'Your photos & display name is how others see you. Customize with special character, spaces, emojis, whatever!',
+    'Your photos & Display name is how others see you. Customize with special character, spaces, emojis, whatever!',
```

Comment:
should this either be display name or Display Name?

Reply:
also if spaces and emojis are pluralized, so too should be special characters

Reply:
Excellent